### ISSUE TITLE
Add support for `--delay`

### ISSUE THREAD

**Issue by twolfson** [twolfson/record-a-cast#2]
Sometimes we want to get ready before we start recording. Let's add a `--delay` which counts down via `console.log` to indicate when we are going to start recording. Prob do it in milliseconds for more control.

``` bash
# Wait 7 seconds before recording
record-a-cast --delay 7000 out.mov
```

**Comment by twolfson** [comment]
Don't forget to add an example to the README

**Comment by twolfson** [comment]
Released in `1.3.0`. Decided we don't need an example
